commons/src/store/contacts.ts

```diff
@@ -36,21 +36,24 @@ function initializeContacts() {
     subscribe,
     addContacts: async (
       query: ContactsQuery,
-      offset: number,
-      limit: number,
+      params: {
+        offset: number;
+        limit: number;
+        group?: string;
+      },
```

Comment:
Ditto here re: changing this to `queryString: string` or `queryString: <new ContactsSearchQuery> interface`